api/proto/teleport/legacy/types/types.proto

```diff
@@ -1562,6 +1562,10 @@ message ClusterNetworkingConfigSpecV2 {
     (gogoproto.jsontag) = "proxy_ping_interval,omitempty",
     (gogoproto.casttype) = "Duration"
   ];
+
+  // CommandExecutionWorkers determines the number of workers that will
+  // execute arbitrary remote commands on servers (e.g. through Assist) in parallel
+  int64 CommandExecutionWorkers = 11 [(gogoproto.jsontag) = "command_execution_workers,omitempty"];
```

Comment:
We can stash it under `auth_service.assist` in YAML. Do you think we can still persist it to ClusterNetworkingConfig (as we want to have a single source of truth for this)?

Reply:
I've stashed the field under Assist config and kept the logic persisting it to the networking config. Please let me know what you think about this solution.

I also introduced errgroup instead of the ad-hoc waitgroup+semaphore.

Reply:
Seems reasonable to me, thanks.